_data/rules.yml

```diff
@@ -37,6 +40,12 @@ permitted:
 - description: This license provides an express grant of patent rights from the contributor to the recipient.
   label: Patent Grant
   tag: patent-grant
+- description: This software may be bundled together with any other software.
```

Comment:
What's the difference between bundling, embedding, and modification/redistribution?